dask_cuda/benchmarks/local_cudf_shuffle.py

```diff
@@ -171,12 +156,34 @@ def main(args):
     print_separator(separator="=")
     print_key_value(key="Wall clock", value="Throughput")
     print_separator(separator="-")
+    t_p = []
+    times = []
     for idx, (data_processed, took) in enumerate(took_list):
         throughput = int(data_processed / took)
         m = format_time(took)
+        times.append(took)
+        t_p.append(throughput)
         print_key_value(key=f"{m}", value=f"{format_bytes(throughput)}/s")
         t_runs[idx] = float(format_bytes(throughput).split(" ")[0])
+    t_p = np.asarray(t_p)
+    times = np.asarray(times)
+    bandwidths_all = np.asarray(bandwidths_all)
     print_separator(separator="=")
+    print_key_value(
+        key="Throughput (Total)",
+        value=f"{format_bytes(t_p.mean())} +/- {format_bytes(t_p.std())}",
+    )
+    print_key_value(
+        key="Throughput (Comms only)",
+        value=f"{format_bytes(bandwidths_all.mean())} +/- "
```

Comment:
Whereas this bandwidth measure is the mean of the all the p2p data movement observeed by dask?

Reply:
Correct.